.circleci/config.yml

```diff
@@ -23,10 +28,58 @@ jobs:
           $job = start-job {dotnet run --project C:\Users\circleci\project\MockServer\MockServer}
           Start-Sleep 10
           .\Tests\doc-downloader.tests.ps1
+          .\Tests\drive-axle-doc-downloader.tests.ps1
       - run: |
           Write-Host (Split-Path -Path $PSScriptRoot)
           $ProgressPreference = 'SilentlyContinue';
           Compress-Archive -Path "C:\Users\circleci\project\README.txt", "C:\Users\circleci\project\configurations.json", "C:\Users\circleci\project\*.ps1" -DestinationPath document-downloader.zip
       - store_artifacts:
           path: document-downloader.zip
           destination: document-downloader.zip
+
+jobs:
+  build_powershell_five:
+    executor:
+      name: win/default
+      shell: powershell.exe
```

Comment:
I'm assuming this relies on PS5 being the default on win/default. I would propose versioning win/default _or_ adding a step to make this explicitly fail if/when the CI base image gets updated past PS5.

Reply:
I can add the win/default version, but is it a real worry since PS 5.1 is the last version of Windows Powershell (ie version of Windows Powershell are no longer being created/updated)

Reply:
Ohh, maybe I'm missing something. I'm assuming `win/default` will eventually get changed by CI (I know they do this from, e.g. their Linux runners) to be a newer version of Windows. I'm assuming Microsoft will eventually start including Powershell Core with new versions of Windows and not the old Windows Powershell. Is that off base?

Reply:
So I believe that just to be the `executer` name. We specefy the windows version (orb version) at the top: `orbs:
  win: circleci/windows@2.2.0`.

Reply:
OH! Got it. I misunderstood. Thank you for explaining!

I'd probably add a comment (`# If you update this value, be sure to check that build_powershell_five is still getting Powershell 5`) for the next person, and then good 2 merge.